### DISCUSSION TITLE
record a path

### DISCUSSION THREAD

**Discussion by Peppobrasil** [Courseplay/Courseplay_FS22#2610] [Q&A]
Still can't record a route to travel later?

**Comment by Tensuko** [comment]
Still can't and won't as mentioned many times.
Use AutoDrive or Giants to let the driver drive arround the map.

**Comment by Peppobrasil** [comment]
Yes! But sorry: Unload, Reversegear, chaff compaction, etc along the path?

**Reply by Tensuko** [reply]
Unload what? Combine unload in combination with AD no Problem.
Reversegear???
Chaff compaction possible for a long time, check ingame help.

**Reply by Peppobrasil** [reply]
Thanks, I'll try now
